### PR TITLE
Add PIC option

### PR DESCRIPTION
**Pull request by WilliamTambellini**
Just add a PIC option: no change of behavior, default still OFF.

### CODE REVIEW

**Comment by WilliamTambellini** [comment]
@gchatelet  any way to merge that one before the new tag/release ? tks

**Comment by gchatelet** [comment]
> @gchatelet any way to merge that one before the new tag/release ? tks

Sorry I had a pending review that was never sent ...

File: CMakeLists.txt [hunk 1]
```diff
@@ -122,6 +124,7 @@ add_library(cpu_features ${CPU_FEATURES_HDRS} ${CPU_FEATURES_SRCS})
 set_target_properties(cpu_features PROPERTIES PUBLIC_HEADER "${CPU_FEATURES_HDRS}")
 setup_include_and_definitions(cpu_features)
 target_link_libraries(cpu_features PUBLIC ${CMAKE_DL_LIBS})
+set_property(TARGET cpu_features PROPERTY POSITION_INDEPENDENT_CODE ${BUILD_PIC})
```
[the remark this answers is not in the document]
Comment (WilliamTambellini) [reply CMakeLists.txt]: It was on purpose, didnt know if that option was supported by windows. Moving it...

**Comment by gchatelet** [comment]
Thx for working on this @WilliamTambellini !

**Pull request merged**